lib/galaxy/tools/__init__.py

```diff
@@ -1079,6 +1080,24 @@ def _parse_citations(self, tool_source):
                     citations.append(citation)
         return citations
 
+    def _parse_xrefs(self, tool_source):
```

Comment:
This method is unused now, I think you need to merge its implementation into `XmlToolSource.parse_xrefs()` in `lib/galaxy/tool_util/parser/xml.py` .

Reply:
I'am not sure what to do there

Reply:
I think that with the other changes I just suggested, this method can be removed.